tests/test_compatibility.py

```diff
@@ -42,25 +42,32 @@ def test_keys(implementations):
     _restore_keys(umbral2, sk_bytes, pk_bytes)
 
 
-def _create_sk_factory_and_sk(umbral, label):
+def _create_sk_factory_and_sk(umbral, skf_label, key_label):
     skf = umbral.SecretKeyFactory.random()
-    sk = skf.secret_key_by_label(label)
-    return bytes(skf), bytes(sk)
+    derived_skf = skf.secret_key_factory_by_label(skf_label)
+    sk = derived_skf.secret_key_by_label(key_label)
```

Comment:
Should it be `by_label` or `from_label`?

Reply:
That would also be my guess, but I think `from` prefix is widely accepted as a naming convention for [`From<>` trait implementation](https://ricardomartins.cc/2016/08/03/convenient_and_idiomatic_conversions_in_rust).

Reply:
"by_label" is a leftover from PyUmbral 0.1. `from` is indeed used generally for constructors.

Come to think of it, perhaps `make_secret_key()`/`make_secret_key_factory()` (or `derive_`, borrowing from PyUmbral 0.1) would be a better name for these methods. `by_label` in my mind implies that there is some lookup going on and not creation.